### DISCUSSION TITLE
Add/Remove tags while parsing xml

### DISCUSSION THREAD

**Discussion by madhuravbhangay** [NaturalIntelligence/fast-xml-parser#461] [Q&A]
Hello,
We are using fast-xml-parser library to convert an input XML to json, the process data and convert back to xml.
Some input xml has some sections like:
```
<description><p>Some text</p></description>
<description><p><p>Some text</p></p></description>
<description><p><p>Some text</p>Some more text </p></description>

```
We want to either add one "p" tag where there is single "p" tag, or we want to remove double "p" tag. 

Is there any option available to add/remove "p" tags in the xml?
Thanks in advance.

**Comment by amitguptagwl** [comment]
Currently, no. But there was another discussion open to discuss to provide a feature to skip some tags from processing. 

Are you converting XML to  Json or JSON to XML?

**Comment by amitguptagwl** [comment]
This is a very specific requirement. We can probably include a feature to skip tags based on jpath
